### PR TITLE
LAST UPDATED (24.04.2017)

### PR DESCRIPTION
**Pull request by sriker123**
Added the events, founder photos and updated the contact information

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,10 +53,22 @@
...
+        auto: true,
+        auto_controls: true
+    });
+});
 </script>
-<style type="text/css">
-
+<!-- DC Elastislide Carousel Settings -->
 
+<style type="text/css">
```
Comment (kapil08svs) [inline index.html:71]: Remove the extra tags

File: index.html [hunk 2]
```diff
@@ -87,14 +105,26 @@
...
+    width: 100%;
 	
 }
-
   .teamImg{
-  width: 100%;
-  height: 100%;
+ <!--max-width: 100%;
+    height: auto;
+    width: auto\9;--->
```
Comment (kapil08svs) [inline index.html:115]: check the comment tags put... extra "-" are there

File: index.html [hunk 3]
```diff
@@ -166,8 +196,13 @@
   .panel-footer {
       background-color: white !important;
   }
+   h3{
```
Comment (kapil08svs) [inline index.html:199]: create a class and assign... don't override the default elements

File: index.html [hunk 4]
```diff
@@ -294,7 +330,23 @@
       margin-top: -3%;
     }
   }
+  .strong{
```
Comment (kapil08svs) [inline index.html:333]: don't override default elements

File: index.html [hunk 5]
```diff
@@ -453,6 +505,7 @@ <h5 id="tagline"> - Invest in yourself</h5>
       <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
       <li data-target="#myCarousel" data-slide-to="1"></li>
       <li data-target="#myCarousel" data-slide-to="2"></li>
+	   <li data-target="#myCarousel" data-slide-to="3"></li>
```
Comment (kapil08svs) [inline index.html:508]: check the indentation

File: index.html [hunk 6]
```diff
@@ -498,42 +551,62 @@ <h5 id="tagline"> - Invest in yourself</h5>
...
 
 
-<div id="people" class="container-fluid bg-grey" style="text-align:center;">
+<div id="people" style="text-align:center;">
     <h2 class="text-center">OUR TEAM</h2>
     <h3></h3>
     <div class="row">
-    <div class="col-sm-3">
-      <img class="teamImg" src="https://raw.githubusercontent.com/kapil08svs/Revamp/master/rsz_adithya.png">
+    <div class="col-xs-12 col-md-2 col-md-offset-1 col-sm-3">
```
Comment (kapil08svs) [inline index.html:574]: classes assigned are different from other divs of same type

**Review by kapil08svs — CHANGES_REQUESTED** [review]

File: index.html (hunk 6 above)
Comment (sriker123) [reply index.html:574]: bro I didn't understand this

**Pull request closed without merging**